<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmSettingDialog</class>
 <widget class="QDialog" name="AlarmSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>95</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkDaily">
      <property name="text">
       <string>Daily</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkWeekdays">
      <property name="text">
       <string>Weekdays</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkWeekend">
      <property name="text">
       <string>Weekend</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>105</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkMon">
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkTue">
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkWed">
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkThu">
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkFri">
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkSat">
      <property name="text">
       <string>Saturday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkSun">
      <property name="text">
       <string>Sunday</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>95</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
